cms/src/test/resources/test-data/api-specifications/SwaggerCodeGenOpenApiSpecificationJsonToHtmlConverterTest/customised-codegen-v3-generated-spec.html

```diff
@@ -1235,14 +1238,104 @@ <h3>Query parameters</h3>
                                                     </table>
                                                 </div>
 
-                                            <h2>Response</h2>
+                                                    <h2>Response</h2>
+
                                                 <h3>HTTP status: 200</h3>
                                                 <div>A completed search, containing zero, one, or many matching patients.</div>
 
 
                                                         <h4>Body</h4>
                                                     
                                                     
+                                                            <h5 class="body__contenttype">
```

Comment:
I may be missing something but it feels surprising that this large block of content would be added here.

Is this definitely correct?

Reply:
Fixed